packages/devtools_app/lib/src/shared/charts/chart.dart

```diff
@@ -542,6 +542,10 @@ class ChartPainter extends CustomPainter {
       ..dispose();
   }
 
+  /// Returns a [TextPainter] containing [textValue], scaled to [scale], with
+  /// some other styling.
+  ///
+  /// Callers are responsible for disposing the [TextPainter].
```

Comment:
have we done this step in DevTools everywhere we can?

Reply:
Yes. This is a private method, and all three callers dispose the TextPainter.